src/geo/GeoPMTFactoryBase.cc

```diff
@@ -136,6 +136,9 @@ G4VPhysicalVolume *GeoPMTFactoryBase::ConstructPMTs(DBLinkPtr table,
     dir_y = lpos_table->GetDArray("dir_y");
     dir_z = lpos_table->GetDArray("dir_z");
   } else {
+    dir_x = lpos_table->GetDArray("x"); //dummy values needed for pmtinfo.AddPMT
+    dir_y = lpos_table->GetDArray("x");
+    dir_z = lpos_table->GetDArray("x");
```

Comment:
I think this might be confusing to people reading the PMTInfo since there is no indication that the direction is in fact unknown. Ideally we could set the actual, calculated directions in PMTInfo, or failing that, set them to some obvious flag value.